homeassistant/components/zwave_js/light.py

```diff
@@ -112,7 +112,11 @@ def __init__(
 
         # get additional (optional) values and set features
         self._target_value = self.get_zwave_value("targetValue")
-        self._target_color = self.get_zwave_value("targetColor")
+        self._target_color = self.get_zwave_value(
+            "targetColor",
+            CommandClass.SWITCH_COLOR,
+            value_property_key=None,
```

Comment:
The default is None, is this necessary?

Should `add_to_watched_value_ids` be set to `False`?

Reply:
I thought `add_to_watched_value_ids` defaults to false doesn't it?

Reply:
Not unless we live in opposite universes. 😛 

https://github.com/home-assistant/core/blob/f772eab7b77c70919ef2cfb335fcf1bf17a7a53c/homeassistant/components/zwave_js/entity.py#L165-L173